### DISCUSSION TITLE
Node Adapter only generating JS files

### DISCUSSION THREAD

**Discussion by ytsruh** [sveltejs/kit#10151] [Q&A]
Hi all, I'm running an Express API and we've had a requirement to add a really small admin dashboard to it - I'm inclined to use SvelteKit to get the team used to this on a larger project.

However testing out the node-adapter for SveltKit appears to generate two files index.js & handler.js as pert the [docs](https://kit.svelte.dev/docs/adapter-node#custom-server). Our Express API is written in TypeScript and so we can't import these JS files without changing our TS config, which we're reluctant to do.

Is it posible to either:
a) Generate TS files from the Adapter, or
b) Explain how this would be used in a TypeScript project?

Thanks in advance

**Comment by ghostdevv** [comment]
Hey, you can import js files into TypeScript. What errors are you running into from ts when trying to do this?

**Reply by ytsruh** [reply]
Thanks for the reply. We're getting this error:
<img width="1417" alt="Screenshot 2023-06-13 at 16 14 11" src="https://github.com/sveltejs/kit/assets/46472228/ce5b05e9-6337-4592-a0c6-a0e348efddad">
